<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MantisDetails</class>
 <widget class="QWidget" name="MantisDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="projectLabel">
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="projectText">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="priorityLabel">
     <property name="text">
      <string>Priority</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="priorityCombo"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="severityLabel">
     <property name="text">
      <string>Severity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="severityCombo"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="reproLabel">
     <property name="text">
      <string>Reproducibility</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="reproCombo"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="statusCombo"/>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="resolutionCombo"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="resolutionLabel">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="versionLabel">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="versionText">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="categoryLabel">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="categoryText">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
